Makefile

```diff
@@ -197,7 +197,14 @@ bundle-cleanup: operator-sdk ## Remove bundle installed via bundle-run
 
 .PHONY: create-ns
 create-ns: ## Create namespace
-	$(KUBECTL) get ns $(BUNDLE_RUN_NAMESPACE) 2>&1> /dev/null || $(KUBECTL) create ns $(BUNDLE_RUN_NAMESPACE)
+	$(KUBECTL) get ns $(BUNDLE_RUN_NAMESPACE) 2>&1> /dev/null || \
```

Comment:
I have modified the var name [_BUNDLE_RUN_NAMESPACE_ to _OPERATOR_NAMESPACE_](https://github.com/medik8s/self-node-remediation/commit/4689318f2218803aad755f98bedf89d3707d9107), and it is used in e2e test (similarly to other Medik8s operators)

Reply:
This why there is a conflict at the moment :)

Reply:
Yep, this was an actual WIP :D